CHANGELOG.md

```diff
@@ -1,5 +1,47 @@
 # [Versions](https://mui.com/versions/)
 
+## 5.13.5
+
```

Comment:
```suggestion

_Jun 12, 2023_

```

Fixed in https://github.com/mui/material-ui/commit/fe2203179e1737ef04ef781f2c749b076758c8aa